ta4j-core/src/main/java/org/ta4j/core/num/DoubleNum.java

```diff
@@ -24,6 +24,8 @@ this software and associated documentation files (the "Software"), to deal in
 
 import java.util.function.Function;
 
+import org.ta4j.core.TestUtils;
```

Comment:
TestUtlis in main package? Should be in test

Reply:
Should not be possible to use imports of test classes in the core package.

Reply:
Looks like everything in test is in package org.ta4j.core or a sub-package of that.  If that changes to org.ta4j.test (or org.ta4j.test.core) then that's probably a separate PR.

Reply:
I think there must be something wrong in your branch? 
![grafik](https://user-images.githubusercontent.com/19409405/36680670-b6c8a51a-1b16-11e8-9d5d-bb343823d4a4.png)

Your IDE should not be able to find TestUtils from core package:
![grafik](https://user-images.githubusercontent.com/19409405/36680601-7f88ffb4-1b16-11e8-8813-9f3fc3fe30a7.png)

Reply:
You will not be able to compile. Travis tests failed because of this import:

``` 
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.7.0:compile (default-compile) on project ta4j-core: Compilation failure: Compilation failure: 
[ERROR] /home/travis/build/ta4j/ta4j/ta4j-core/src/main/java/org/ta4j/core/num/DoubleNum.java:[27,21] cannot find symbol
[ERROR]   symbol:   class TestUtils
[ERROR]   location: package org.ta4j.core
[ERROR] /home/travis/build/ta4j/ta4j/ta4j-core/src/main/java/org/ta4j/core/num/DoubleNum.java:[42,39] cannot find symbol
[ERROR]   symbol:   variable TestUtils
[ERROR]   location: class org.ta4j.core.num.DoubleNum
```